46-lightning-detector/README.md

```diff
@@ -0,0 +1,185 @@
+
+```json
+{
+    "type": 2,
+    "distance_to_storm": 15,
+    "energy": 999
+}
+```
+
+- `type` is a code to indicate the event type. 0 is noise, 1 is disturber, and 2 is lightning.
+- `distance_to_storm` is an additional value reported for lightning events. It's the distance to the head of the storm in kilometers.
+- `energy` is an additional value reported for lightning events. According to the datasheet:
+
+  >This value is just a pure number and has no physical meaning.
+
+There are also a handful of optional [environment variables](https://dev.blues.io/guides-and-tutorials/notecard-guides/understanding-environment-variables/) that you can tweak to alter the characteristics of the lightning detector. These will be covered in detail in a later section.
+
+## Hardware Setup
+
+### AS3935
+
+Break off a row of 7 headers and [solder](https://learn.sparkfun.com/tutorials/how-to-solder-through-hole-soldering/all) them into the holes on the AS3935 breakout board.
+
+### Notecarrier
+
+1. Assemble Notecard and Notecarrier as described in the [Notecard Quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/).
+2. Plug the Swan into the Notecarrier, aligning the Swan's male headers with the Notecarrier's female headers.
+3. Using jumper wires, make the following connections between the detector board and the Notecarrier:
+    | Detector | Notecarrier |
+    | ---------| ----------- |
+    | MOSI     | F_MOSI      |
+    | MISO     | F_MISO      |
+    | SCK      | F_SCK       |
+    | CS       | F_D5        |
+    | INT      | F_D6        |
+    | 3V3      | F_3V3       |
+    | GND      | GND         |
+4. Connect the Swan to your development PC with the micro USB cable. The LED on the detector board will turn on.
+  <p align="center">
+  <img src="images/assembled.jpg"/>
+  </p>
+
+## Notehub Setup
+
+Sign up for a free account on [Notehub](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+### Environment Variables
+
+There are several optional [environment variables](https://dev.blues.io/guides-and-tutorials/notecard-guides/understanding-environment-variables/) that change the detection behavior of the lightning detector. You can set these values for your device from Notehub:
+
+1. From your project's main page, click Devices.
+2. Double-click your device in the list.
+3. Click the Environment tab.
+4. Click the "New key" textbox in the Device section and enter the environment variable and fill in the desired value.
+5. Click "Apply changes".
+
+Here are the environment variables supported by the firmware:
+
+- `ignore_disturbers`: If this is 0, disturbers trigger the creation of an event Note. If this is 1, disturbers do NOT create an event Note.
+
+  *Default: 0*
```

Comment:
The source formatting is a little strange with the blank lines before the default values, although it renders just fine.